### DISCUSSION TITLE
failed to resolve import virtual:uno.css from 'src\main.tsx'，does the file exist?

### DISCUSSION THREAD

**Discussion by ylh1103** [unocss/unocss#4360] [Q&A]
node版本14，vite版本4，在安装后引入virtual:uno.css启动报错failed to resolve import virtual:uno.css from 'src\main.tsx'，does the file exist? Plugin: vite:importanalysis
是因为我的环境版本太低吗，请问哪个版本可以适用这个环境呢

**Comment by zyyv** [comment]
The node version is too low, UnoCSS requires the minimum node >= 18 version.

**Reply by ylh1103** [reply]
ok，thanks
